app/org/sagebionetworks/bridge/models/DateTimeRangeResourceList.java

```diff
@@ -10,31 +10,24 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
-public class DateTimeRangeResourceList<T> {
-    private final List<T> items;
-    private final DateTime startTime;
-    private final DateTime endTime;
-
+public class DateTimeRangeResourceList<T> extends ResourceList<T> {
+    
     @JsonCreator
-    public DateTimeRangeResourceList(@JsonProperty("items") List<T> items,
-            @JsonProperty("startTime") DateTime startTime, @JsonProperty("endTime") DateTime endTime) {
-        this.items = items;
-        this.startTime = startTime;
-        this.endTime = endTime;
-    }
-    public List<T> getItems() {
-        return items;
-    }
-    public int getTotal() {
-        return items.size();
+    public DateTimeRangeResourceList(@JsonProperty(ITEMS) List<T> items) {
+        super(items);
     }
+    @Deprecated
     @JsonSerialize(using = DateTimeSerializer.class)
     public DateTime getStartTime() {
-        return startTime;
+        return getDateTime(START_TIME);
     }
+    @Deprecated
     @JsonSerialize(using = DateTimeSerializer.class)
     public DateTime getEndTime() {
-        return endTime;
+        return getDateTime(END_TIME);
+    }
```

Comment:
These are tested in serialization... added tests that these are retrievable from the deprecated getters in the object.